spec/views/posts/index.html.erb_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "posts/index", type: :view do
+  before(:each) do
+    assign(:posts, [
+      Post.create!(
+        title: "Title",
+        body: "MyText"
+      ),
+      Post.create!(
+        title: "Title",
+        body: "MyText"
+      )
+    ])
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstArrayElementIndentation: Indent the right bracket the same as the first position after the preceding left parenthesis.